tests/test_mathematical.py

```diff
@@ -208,10 +208,6 @@ def test_minimum(self, dtype, lhs, rhs):
     def test_multiply(self, dtype, lhs, rhs):
         self._test_mathematical("multiply", dtype, lhs, rhs)
 
-    @pytest.mark.skipif(
-        not has_support_aspect64(), reason="Aborted on Iris Xe: SAT-6039"
-    )
-    @pytest.mark.usefixtures("allow_fall_back_on_numpy")
     @pytest.mark.parametrize(
         "dtype", get_all_dtypes(no_bool=True, no_complex=True)
     )
```

Comment:
test failed on Iris Xe, since
```python
a = numpy.array([6, 3], dtype='i4')
a % 0.3
# Out: array([2.22044605e-16, 1.11022302e-16])

b = dpt.asarray([6, 3], dtype='i4')
b % 0.3
# Out: usm_ndarray([0.29999977, 0.2999999 ], dtype=float32)
```
I wonder if it's expected or the issue in dpctl.

Reply:
I noticed it and I think it is expected, the output of numpy is `float64` and output of dpnp is `float32`
If I write the numpy example as
`numpy.remainder(numpy.array([6, 3], dtype='i4'), 0.3, dtype='f4')` 
I get similar results to dpnp
`# Out: array([0.29999977, 0.2999999 ], dtype=float32)`

Reply:
Ok, then it needs to be handled in tests somehow. As an option we can skip unaligned tests on Iris Xe.